src/ocm/components/clusters/ClustersTable.tsx

```diff
@@ -28,17 +31,31 @@ interface ClustersTableProps {
   deleteCluster: (id: string) => void;
 }
 
-const columnConfig = {
+type TablePropsCellType = TableProps['cells'][0];
```

Comment:
Maybe we should take this type to a commonplace, so it can be reused later by any other component that consumes the Table from PF.